CHANGELOG.md

```diff
@@ -3,6 +3,103 @@ FEATURES:
 * CLI
   * The `consul-k8s` CLI enables users to deploy and operate Consul on Kubernetes.
     * Support `consul-k8s install` command. [[GH-713](https://github.com/hashicorp/consul-k8s/pull/713)]
+* Helm Chart
+  * Add support for Admin Partitions. **(Consul Enterprise only)**
+  **ALPHA** [[GH-729](https://github.com/hashicorp/consul-k8s/pull/729)]
+
+This feature allows Consul to be deployed across multiple Kubernetes clusters while sharing a single set of Consul
```

Comment:
I think this section needs to be intended so that it all looks like a part of the admin partitions bullet point.